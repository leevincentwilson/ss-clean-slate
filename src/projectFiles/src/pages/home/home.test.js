import { configure, shallow } from 'enzyme';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Home from './index';

configure({ adapter: new Adapter() });

describe('<Home />', () => {
  test('It should render the home page', () => {
    const wrapper = shallow(<Home />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
