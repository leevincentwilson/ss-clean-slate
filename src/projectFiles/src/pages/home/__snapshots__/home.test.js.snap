// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> It should render the home page 1`] = `
<div>
  Hello World!
</div>
`;
